autocut/utils.py

```diff
@@ -2,8 +2,23 @@
 import os
 import re
 
-import srt
+import ffmpeg
+import numpy as np
 import opencc
+import srt
+
+
+def load_audio(file: str, sr: int = 16000) -> np.ndarray:
```

Comment:
This step can be quite expensive and memory intensive? Is it possible to directly give the file to local/openai whisper? both of them should have that option?

Reply:
Vad is a pre selectable step for all model inference, requiring the reading of audio files